### PR TITLE
Implement #10: 'Block user edit values given by OpenID.'

### PR DESCRIPTION
**Pull request by tekhnee**

### CODE REVIEW

File: css/server.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#security-password {
```
Comment (pulsejet) [inline css/server.css:1]: server.css isn't a very appropriate name. Often can make a difference when debugging (don't know nextcloud specifics)

File: lib/AppInfo/Application.php [hunk 2]
```diff
@@ -79,6 +80,7 @@ public function register()
                     exit();
                 });
             }
+	    Util::addStyle($this->appName, 'server');
```
Comment (pulsejet) [inline lib/AppInfo/Application.php:83]: Is this indent correct? Also this needs to be configurable.

**Review by pulsejet — COMMENTED** [review]
The relevant issue is #26 not  #10

File: css/server.css (hunk 1 above)
Comment (tekhnee) [reply css/server.css:1]: Actually, `server.css` is the name NC expects to find theme-level CSS in. This app not being a theme, it probably doesn't need to adhere to the convention.

File: lib/AppInfo/Application.php (hunk 2 above)
Comment (tekhnee) [reply lib/AppInfo/Application.php:83]: The indent is a typo—sorry for missing the target.

Unfortunately more urgent priorities interfere, so I won't be able to add a UI to this — very sorry… Hopefully you or some other contributor might pick it up from here.

Comment (pulsejet) [reply lib/AppInfo/Application.php:83]: I'll pick it up don't worry. No UI, btw. The whole plugin's config is deliberately done through the global config.

Comment (tekhnee) [reply lib/AppInfo/Application.php]: I actually did it, no worries. Pls allow me a few minutes to update.

Comment (tekhnee) [reply lib/AppInfo/Application.php]: Continued as PR #60.

**Pull request closed without merging**